ci/upload_tarball_to_release.sh

```
@@ -1,16 +1,17 @@
 #!/usr/bin/env bash
 
+lfortran_version=$(<version)
```

Above this you must put `set -ex`. Otherwise it ignores errors.